app/helpers/home_helper.rb

```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```

Comment:
Kill it, there are a lot of files that are generated with Rails command line tools like `rails generate` and `rails scaffold`, some are useful, others are not (like feature specific js, css, and Helper modules). For the ones that you don't immediately find useful, keeping them around just adds fluff to your code base.